kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue

```diff
@@ -108,18 +106,24 @@
       <p v-if="!exams.length">
         {{ $tr('noExams') }}
       </p>
-      <!--       <p
-          v-else-if="statusSelected.value === coachString('activeQuizzesLabel') &&
-            !activeExams.length"
-        >
-          {{ $tr('noActiveExams') }}
-        </p>
-        <p
-          v-else-if=" statusSelected.value === coachString('inactiveQuizzesLabel') &&
-            !inactiveExams.length"
-        >
-          {{ $tr('noInactiveExams') }}
-        </p> -->
+      <p
+        v-else-if="statusSelected.value === coachString('filterQuizStarted') &&
+          !activeExams.length"
+      >
+        {{ $tr('noActiveExams') }}
```

Comment:
Not sure what exactly is this condition supposed to do, but filtering by _active exams_ makes no sense any more...

Reply:
@radinamatic  These are specific messages displayed if no exams are returned on filtering. The confusion arises(apologies on the confusion!) with the names of the translation strings. These have been refactors to reflect started, not started and ended.